Cargo.toml

```diff
@@ -11,7 +11,10 @@ readme = "README.md"
 license = "MIT"
 
 [features]
+default = []
 stats = []
+datasets = []
+test = []
```

Comment:
We don't need to include the `test` or `default` features. These already exist as defined here.